src/main/drivers/bus.h

```diff
@@ -19,14 +19,21 @@
 
 #include "platform.h"
 
+#include "drivers/bus_i2c.h"
 #include "drivers/io_types.h"
 
 typedef union busDevice_u {
```

Comment:
Yes, I'm thinking along similar lines. However I don't favour adding it now - rather we should add it when we convert first dual interface device.